### DISCUSSION TITLE
nuxt3 document website is broken

### DISCUSSION THREAD

**Discussion by KahrLu** [nuxt/nuxt#17406] [Questions]
![image](https://user-images.githubusercontent.com/76502123/174774393-05a9f65a-1211-4c82-a23e-d7c7cfb50dbe.png)
@pi0

**Comment by pi0** [comment]
Hi @KahrLu. It seems caused by a Cloudflare accident (https://www.cloudflarestatus.com/incidents/xvs51y9qs9dj). Can you please try hard refreshing or using an incognito or another browser if still having issues?

**Reply by KahrLu** [reply]
> It seems caused by a Cloudflare accident (https://www.cloudflarestatus.com/incidents/xvs51y9qs9dj). Can you please try hard refreshing or using an incognito or another browser if still having issues?

Thanks, No matter which method I try, can't see the normal page. we may have to wait for cloudflare to solve it
